### PR TITLE
[DM] BTAT-5656 added field to model

### PR DESCRIPTION
**Pull request by TheDeadlyPianist**
These don't necessarily need doing first, but the following also need merging:
- [ ] [App-Config-Base](https://github.com/hmrc/app-config-base/pull/2091)
- [ ] [App-Config-QA](https://github.com/hmrc/app-config-qa/pull/3869)
- [ ] [App-Config-Prod](https://github.com/hmrc/app-config-production/pull/3576)

### CODE REVIEW

File: it/uk/gov/hmrc/vatsubscription/controllers/RetrieveVatKnownFactsControllerISpec.scala [hunk 1]
```diff
@@ -54,6 +55,10 @@ class RetrieveVatKnownFactsControllerISpec extends ComponentSpecBase with Before
   )
 
   "GET /:vatNumber/known-facts" when {
+
```
[the remark this answers is not in the document]
Comment (TheDeadlyPianist) [reply it/uk/gov/hmrc/vatsubscription/controllers/RetrieveVatKnownFactsControllerISpec.scala]: Not even sure how all of that got there.

File: app/uk/gov/hmrc/vatsubscription/models/VatCustomerInformation.scala [hunk 2]
```diff
@@ -135,6 +139,7 @@ object VatCustomerInformation extends JsonReadUtil with JsonObjectSugar {
     model =>
       jsonObjNoNulls(
         "mandationStatus" -> model.mandationStatus.value,
+        "overseasIndicator" -> model.customerDetails.overseasIndicator,
```
[the remark this answers is not in the document]
Comment (TheDeadlyPianist) [reply app/uk/gov/hmrc/vatsubscription/models/VatCustomerInformation.scala]: The above one has Customer Details, which already contains this.

File: README.md [hunk 3]
```diff
@@ -152,7 +154,7 @@ Where:
 ##### UNEXPECTED_GET_VAT_CUSTOMER_INFORMATION_FAILURE
 * **Status**: (any)
 
-### GET /vat-subscription/:vatNumber/manage-account-summary
+### GET /vat-subscri ption/:vatNumber/manage-account-summary
```
Comment (TomDudley92) [inline README.md]: You've added a space in subscription 🙀

File: app/uk/gov/hmrc/vatsubscription/config/featureSwitch/FeatureSwitchModel.scala [hunk 4]
```diff
@@ -25,4 +25,4 @@ case class FeatureSwitchModel(latestApi1363Version: Boolean,
 
 object FeatureSwitchModel {
   implicit val format: OFormat[FeatureSwitchModel] = Json.format[FeatureSwitchModel]
-}
\ No newline at end of file
+}
```
Comment (TomDudley92) [inline app/uk/gov/hmrc/vatsubscription/config/featureSwitch/FeatureSwitchModel.scala]: If you've removed the latestApi1363 from config do we still need it here in the Feature Switch Model?

File: app/uk/gov/hmrc/vatsubscription/connectors/UpdateVatSubscriptionConnector.scala [hunk 5]
```diff
@@ -41,6 +41,7 @@ class UpdateVatSubscriptionConnector @Inject()(val http: HttpClient,
     case Api1365R5 => DESApi1365WritesR5
     case Api1365R6 => DESApi1365WritesR6
     case Api1365R7 => DESApi1365WritesR7
+
```
Comment (TomDudley92) [inline app/uk/gov/hmrc/vatsubscription/connectors/UpdateVatSubscriptionConnector.scala]: Do we still need the older reads? Or do you think this could be scope creep if you remove this as well?

File: test/uk/gov/hmrc/vatsubscription/controllers/RetrieveVatCustomerDetailsControllerSpec.scala [hunk 6]
```diff
@@ -22,6 +22,7 @@ import play.api.libs.json.Json
 import play.api.mvc.Result
 import play.api.test.FakeRequest
 import uk.gov.hmrc.auth.core.InsufficientEnrolments
+import uk.gov.hmrc.vatsubscription.config.featureSwitch.{Api1363R10, Api1363R8}
```
Comment (TomDudley92) [inline test/uk/gov/hmrc/vatsubscription/controllers/RetrieveVatCustomerDetailsControllerSpec.scala]: Unused import here

File: app/uk/gov/hmrc/vatsubscription/connectors/UpdateVatSubscriptionConnector.scala (hunk 5 above)
Comment (TheDeadlyPianist) [reply app/uk/gov/hmrc/vatsubscription/connectors/UpdateVatSubscriptionConnector.scala]: Scope creep I imagine. I think we need an extra story or tech to just go through and remove them. I have no idea how much other stuff is linked to them.

Comment (TheDeadlyPianist) [reply app/uk/gov/hmrc/vatsubscription/connectors/UpdateVatSubscriptionConnector.scala]: I'll remove it and see if any problems rise up. If it's not something super fast though, I'll avoid it.
